src/_helpers/typography.ts

```diff
@@ -220,4 +229,5 @@ export const typography = {
   body16,
   body14,
   subheading,
+  subheadingLarge,
```

Comment:
That’s a new one. Did we have 2 subheading sizes previously? Or is this a new pattern?

Reply:
We've had both in the past. So far we only had the small one in the header for tetra. On pages, it uses 13px, I switched it to 14 since we don't have 13px as a font size in tetra.